ephios/settings.py

```diff
@@ -68,7 +69,7 @@
     "ephios.extra",
     "ephios.api",
 ]
-INSTALLED_APPS += EPHIOS_CORE_MODULES
+INSTALLED_APPS = EPHIOS_CORE_MODULES + INSTALLED_APPS
```

Comment:
Why change the ordering?

Reply:
oauth2 allows to change the template, but the way they want us to do that is by overriding the oauth2_provider/*.html templates. the first match is used by django, so with the old ordering django always served the built-in template

Reply:
I see, thanks. Maybe put a comment so we wont break it when touching this in the future?

Reply:
something in the back of my head said that another dependency required us to import it _after_ the dependency, but I couldn't remember which one (my head said dynmic-preferences, but that wasn't true). As tests still work I would suggest going with this PR and coming back to it if something breaks